cmd/collector/app/processor/constants.go

```diff
@@ -30,6 +30,8 @@ const (
 	JaegerSpanFormat SpanFormat = "jaeger"
 	// ZipkinSpanFormat is for Zipkin Thrift spans.
 	ZipkinSpanFormat SpanFormat = "zipkin"
+	// KafkaSpanFormat is for kafka spans.
+	KafkaSpanFormat SpanFormat = "kafka"
```

Comment:
Kafka is a transport, not a span format

Reply:
```suggestion
	KafkaTransport InboundTransport = "kafka"
```
Something like this?